### PR TITLE
ENGDOCS-2073

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Add info on the new proxy support items for DD 4.30

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/desktop/settings/windows.md [hunk 1]
```diff
@@ -216,6 +212,25 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 If you are running Windows containers in Docker, you can allow the Windows Docker daemon to use Docker Desktop's internal proxy, with the **Use proxy for Windows Docker daemon** setting.
 This is useful when a corporate proxy that requires authentication is manually configured or set at the system level. If you are an admin for your organization and have a Docker Business subscription, you can control this setting with [Settings management](../hardened-desktop/settings-management/configure.md) using the `windowsDockerdPort` parameter.
 
+#### Proxy authentication
+
+If the proxy requires authorization then Docker Desktop dynamically asks
+the developer for a username and password. All passwords are stored securely in the OS credential store.
```
Comment (dvdksn) [inline content/desktop/settings/windows.md:218]:
```suggestion
If the proxy requires authentication then Docker Desktop dynamically asks
the developer for a username and password. All passwords are stored securely in the OS credential store.
```

I also wonder about the phrase "dynamically asks/prompts", here and in the basic auth section below. I guess it would be better to say that basic auth triggers an authentication prompt (and how/when it's triggered)

File: content/desktop/settings/windows.md [hunk 2]
```diff
@@ -216,6 +212,25 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
...
+
+Docker Desktop supports Basic, Kerberos and NTLM proxy authentication methods. 
+
+##### Basic authentication
+
+If your proxy uses Basic authentication, Docker Desktop dynamically prompts developers for a username and password. 
+
+It's recommended that you use an `https:// URL` for HTTP/HTTPS proxies to protect passwords during network transit. Docker Desktop also supports TLS 1.3 for communication with proxies.
```
Comment (dvdksn) [inline content/desktop/settings/windows.md:226]:
```suggestion
It's recommended that you use an `https://` URL for HTTP/HTTPS proxies to protect passwords during network transit. Docker Desktop also supports TLS 1.3 for communication with proxies.
```

File: content/desktop/settings/windows.md [hunk 3]
```diff
@@ -216,6 +212,25 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
...
+
+##### Kerberos and NTLM authentication
+
+Kerberos and NTLM proxy authentication are available for Pro, Team, and Business subscribers with Docker Desktop version 4.30 and later. No additional configuration is needed beyond specifying the proxy IP address and port.
+
+You can enjoy a seamless experience without being interrupted by prompts for proxy credentials, reducing the risk of account lockouts due to incorrect sign in attempts.
```
Comment (dvdksn) [inline content/desktop/settings/windows.md:232]: I'd rephrase this sentence to describe the why as well. i.e. with NTLM/Kerberos, the user isn't prompted for auth to the proxy because with these methods authentication is centralized (or something of that nature)

File: content/desktop/settings/windows.md [hunk 4]
```diff
@@ -216,6 +212,22 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
...
+
+##### Basic authentication
+
+If your proxy uses Basic authentication, Docker Desktop prompts developers for a username and password and caches the credentials. All passwords are stored securely in the OS credential store. It will request re-authentication if that cache is removed.
+
+It's recommended that you use an `https://` URL for HTTP/HTTPS proxies to protect passwords during network transit. Docker Desktop also supports TLS 1.3 for communication with proxies.
+
+##### Kerberos and NTLM authentication
+
+Kerberos and NTLM proxy authentication are available for Pro, Team, and Business subscribers with Docker Desktop version 4.30 and later. No additional configuration is needed beyond specifying the proxy IP address and port.
```
Comment (karman-docker) [inline content/desktop/settings/windows.md:227]: @aevesdocker @dvdksn I provided this content, but correct me if I am wrong,  should we say 'Docker Desktop release 4.30' instead of version 4.30 ?

Comment (karman-docker) [reply content/desktop/settings/windows.md:227]: we refer to it as version in other places of our docs (https://docs.docker.com/desktop/mac/permission-requirements/#installing-symlinks)). So let's keep it as is.

**Review by karman-docker — APPROVED** [review]

File: content/desktop/settings/windows.md (hunk 4 above)
Comment (karman-docker) [inline content/desktop/settings/windows.md:227]: @aevesdocker  as per the discussion here: https://docker.slack.com/archives/C0W4XJVFW/p1714486276644979?thread_ts=1714127067.533579&cid=C0W4XJVFW

there has been a change to the required subscription level for Kerberos and NTLM authentication. Can we update it to:
"Kerberos and NTLM proxy authentication are available for Business subscribers with Docker Desktop version 4.30 and later."

File: content/desktop/settings/windows.md [hunk 5]
```diff
@@ -216,6 +212,22 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 If you are running Windows containers in Docker, you can allow the Windows Docker daemon to use Docker Desktop's internal proxy, with the **Use proxy for Windows Docker daemon** setting.
 This is useful when a corporate proxy that requires authentication is manually configured or set at the system level. If you are an admin for your organization and have a Docker Business subscription, you can control this setting with [Settings management](../hardened-desktop/settings-management/configure.md) using the `windowsDockerdPort` parameter.
 
+#### Proxy authentication
+
+Docker Desktop supports Basic, Kerberos and NTLM proxy authentication methods.
```
Comment (usha-mandya) [inline content/desktop/settings/windows.md:217]: Nit

```suggestion
Docker Desktop supports Basic, Kerberos, and NTLM proxy authentication methods. 
```

**Review by usha-mandya — DISMISSED** [review]
LGTM

**Review by stephaurelio — APPROVED** [review]
LGTM
